packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -28,15 +31,27 @@ const BuySellForm = props => {
         min_order_amount_limit_display,
         payment_method_names,
         price,
+        rate_type,
     } = buy_sell_store?.advert || {};
 
     const style = {
         borderColor: 'var(--brand-secondary)',
         borderWidth: '2px',
     };
 
+    const fetchCurrentMarketRate = async () => {
+        const response = await requestWS({
+            exchange_rates: 1,
+            base_currency: currency,
+            target_currency: local_currency_config.currency,
+        });
+        const { rates } = response.exchange_rates;
```

Comment:
Shouldn't we add a check for `response` here?

Reply:
Will make the change.